app/lib/bing_api_v7.rb

```diff
@@ -0,0 +1,38 @@
+require 'net/https'
+require 'uri'
+require 'json'
+
+class BingApiV7
+  URI::PATH = "https://api.cognitive.microsoft.com/bing/v7.0/search".freeze
```

Comment:
incorrect constant declaration
add constant to your class instead of defining to a gem
it suppose to to 2 constants, e.g.:
```
class BingApiV7
  BING_URI = 'https://api.cognitive.microsoft.com'.freeze
  BING_PATH = '/bing/v7.0/search'.freeze

# or as one
  BING_API_ENDPOINT = 'https://api.cognitive.microsoft.com/bing/v7.0/search'.freeze;

Reply:
go with 2 constants 
 BING_URI = 'https://api.cognitive.microsoft.com'.freeze
  BING_PATH = '/bing/v7.0/search'.freeze

Never add constants to existing classes, like URI::PATH. In 99.999999999% cases it's an issue. Always create OWN constants